config/storageClasses.yaml

```diff
@@ -50,7 +52,7 @@ storageClasses:
       visitInfo: TablePersistableVisitInfo
       apCorrMap: TablePersistableApCorr
       coaddInputs: TablePersistableCoaddInputs
-      metadata: PropertyList
+      metadata: PropertySet
```

Comment:
Huh.  Just an FYI: I was under the impression that the thing returned by `Exposure.getMetadata()` was always a `PropertyList`, but it turns out that's because it's implemented that way and pybind11 always downcasts to the most-derived type; the C++ does say `PropertySet`.  I'm inclined to try to get that changed in afw, because I think we really do depend on it being a `PropertyList` in various places, so this will probably get reverted eventually.